### PR TITLE
Validations

### PR DESCRIPTION
**Pull request by Rexben001**
## Description
Validate input fields when creating items
Limit the quantities per item to 20
Trim whitespaces
Add a placeholder for phone number
## Type of change
Please delete options that are not relevant.
- [x] Bug fix (non-breaking change which fixes an issue)
## Checklist
Remove any items which are not applicable.
- [x] I have performed a self-review of my own code
- [x] My code follows the style guidelines of this project
- [x] My changes generate no new warnings
## Link to Trello card
https://trello.com/c/LCvZBgkU/187-validations

### CODE REVIEW

**Comment by Rexben001** [comment]
[Validations](https://trello.com/c/LCvZBgkU/187-validations)

File: src/index.js [hunk 1]
```diff
@@ -19,4 +19,4 @@ ReactDOM.render(
 // If you want your app to work offline and load faster, you can change
 // unregister() to register() below. Note this comes with some pitfalls.
 // Learn more about service workers: https://bit.ly/CRA-PWA
-serviceWorker.register()
+serviceWorker.unregister()
```
Comment (aapetsi) [inline src/index.js]: why have you unregistered the service worker?

**Review by aapetsi — COMMENTED** [review]
Good stuff. Left some comments

**Review by shaunorpen — CHANGES_REQUESTED** [review]
Great work, just a few questions / suggestions:

1. Why have you unregistered the service worker? I thought it was required for the PWA to work?
2. Why have you made images mandatory when creating / editing items? I think @DimejiAre is working on a change to make items with no image display the app logo as a placeholder. I don't think we need both updates.
3. Please darken the phone icon on the reset password screen to match the register / login screen.

**Review by shaunorpen — APPROVED** [review]
Great work!

**Pull request merged**